qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -0,0 +1,145 @@
+import os
+import re
+import sys
+import shutil
+
+
+# Usage of win_driver_install.py.
+def usage(script_name):
+    print "Usage of %s:" % script_name
+    use_str = " %s" % script_name
+    use_str += " --op={install_driver|uninstall_driver|query_driver|help}"

```

Comment:
@suqinhuang 

Is this better ?

# python win_driver_install.py --help
usage: win_driver_install.py [-h] [-i] [-u] [-q] [--driver_path DRIVER_PATH]
                             [--driver_name DRIVER_NAME]
                             [--device_name DEVICE_NAME]
                             [--vol_utils VOL_UTILS]

Windows Driver Operation

optional arguments:
  -h, --help            show this help message and exit
  -i, --install_driver  operation for install driver
  -u, --uninstall_driver
                        operation for uninstall driver
  -q, --query_driver    operation for query driver
  --driver_path DRIVER_PATH
                        driver path
  --driver_name DRIVER_NAME
                        driver name
  --device_name DEVICE_NAME
                        the corresponding device name with driver
  --vol_utils VOL_UTILS
                        volume of WIN_UTILS